schema/deploy/insert_todo.sql

```diff
@@ -0,0 +1,14 @@
+-- Deploy todo_app:insert_todo to pg
+-- requires: todos
+-- requires: todo_appschema
+
+BEGIN;
+
+create or replace function todo_app.insert_todo(
```

Comment:
In most cases you won't need to create functions to insert your data. postgraphile will automatically generate graphql mutations to insert, update and delete data.